packages/fxa-payments-server/src/lib/reactga-event.ts

```diff
@@ -0,0 +1,91 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import ReactGA from 'react-ga4';
+import { Plan } from 'fxa-shared/subscriptions/types';
+
+export enum GAEvent {
+  AddPaymentInfo = 'add_payment_info',
+  Purchase = 'purchase',
+  PurchaseSubmit = 'purchase_submit',
+  SignUp = 'sign_up',
+}
+
+export enum GAPaymentType {
+  CreditCard = 'Credit Card',
+  PayPal = 'PayPal',
+  NotChosen = 'Not chosen',
+  Undefined = 'Not chosen',
+}
+
+export enum GAPurchaseType {
+  New = 'new purchase',
+  Upgrade = 'upgrade',
+}
+
+interface ReactGALogProps {
```

Comment:
shouldn't affect your code/be a big change - generally in this cases `type` is more appropriate than `interface`. for functional stuff (as opposed to more OO stuff) or things where you won't need declaration merging, i think `type` is preferred.

i dont think we have any official code style guide around this, so author's choice.

Reply:
Note to self. Thank you! That's good to know.